src/processing/record.py

```diff
@@ -0,0 +1,43 @@
+import cv2
+import keyboard
+
```

Comment:
All data input should be handled in the DataStream interface. If you need to add some functionality, like un-distorting an image, you should take in a distorted image in one of the defined functions(or add a new function) in the [design doc](https://github.com/HelloWorldSeniorProject/AirKeys/blob/develop/documentation/design/output/data_processing.pdf), then modify it as necessary, and return a new object.